### PR TITLE
Hegemony waypoint

### PR DESCRIPTION
**Pull request by hazelrat**
This introduces the Hegemony Waypoint, a hegemony variant of the sensor relay intended to replace the sensor relay in Uueoa-Esa. 

TODO: 

- [x] Fix the overmap object spawning twice for some reason
- [x] Get sensors working
- [x] Introduce grime and more details to map
- [x] Exclude sensor relay from Uueoa-Esa
- [x] Add unique sprite
- [x] Introduced mapmanip elements so there can be randomised and potentially dangerous stuff in the offsite (maybe)

Featuring randomised rooms!

image

### CODE REVIEW

**Comment by hazelrat** [comment]
!wip

**Comment by hazelrat** [comment]
!review

**Comment by DreamySkrell** [comment]
image

for better organization, all sensor relays should be in the same folder: `maps\away\away_site\sensor_relay`

the current generic relay should be in a folder like `maps\away\away_site\sensor_relay\generic` or `\solarian_prewar` or something

and the hegemony waypoint relay should be like `maps\away\away_site\sensor_relay\hegemony_waypoint`

File: maps/away/away_site/sensor_relay/sensor_relay.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	sectors = ALL_POSSIBLE_SECTORS
 	id = "sensor_relay"
 	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
-	sectors_blacklist = list(SECTOR_LEMURIAN_SEA) //because finding a sensor relay in hell would be weird
+	sectors_blacklist = list(SECTOR_LEMURIAN_SEA, SECTOR_UUEOAESA) //because finding a sensor relay in hell would be weird + there is a specific relay variant in uueoa-esa
```
Comment (DreamySkrell) [inline maps/away/away_site/sensor_relay/sensor_relay.dm:13]: image
make it like this to be more readable

File: maps/away/away_site/uueoaesa/hegemony_waypoint/hegemony_waypoint.dm [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+	comms_name = "Hegemony Waypoint"
+	initial_generic_waypoints = list(
+		"waypoint_dock_n",
+		"waypoint_dock_e",
+		"waypoint_dock_w",
+		"waypoint_dock_s",
+		"waypoint_n_space",
+		"waypoint_e_space",
+		"waypoint_w_space",
+		"waypoint_s_space"
```
Comment (DreamySkrell) [inline maps/away/away_site/uueoaesa/hegemony_waypoint/hegemony_waypoint.dm:33]: all of these should be renamed to `hegemony_waypoint_ .... `

**Comment by hazelrat** [comment]
> as above

All changes implemented.

**Comment by DreamySkrell** [comment]
image
image
need to change tags on the landmark defs too

**Comment by hazelrat** [comment]
> image image need to change tags on the landmark defs too

Done!

**Review by DreamySkrell — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**